keps/sig-node/xxx-dynamic-resource-allocation/README.md

```diff
@@ -0,0 +1,1428 @@
+  the node without a device. GPU and crypto-offload engines are
+  examples of this kind of device. If they’re not available, workloads
+  can still run by falling back to using only the CPU for the same
+  task.
+
+  *Limitation*: Optional allocation is supported neither by the device
+  plugins nor by current Pod resource declaration.
+
+- *Support Over the Fabric devices*: When deploying a container, I’d
+  like to utilize devices available over the Fabric (PCIe, CXL,
+  Network, special links, etc).
+
+  *Limitation*: Device Plugins framework supports only local devices.
+
+- *Access to the container image*: When deploying a container that
+  needs access to a GPU device, I would like my container to
+  gracefully fail to start (rather than SIGSEGV at runtime) if a
+  minimum driver requirement is not met on the host.
+
+  *Limitation*: GPU driver requirements are currently stored as labels
+  on the container image, and device plugins do not have access to the
+  container image.
+
+Because this KEP enables the usage of
+[CDI](https://github.com/container-orchestrated-devices/container-device-interface/#readme)
+in Kubernetes, it also addresses those problems that are handled by
+CDI:
+
+- *Perform container runtime specific operations*: When deploying a container
+  that needs access to a device, I would like to be able to reuse the
+  same pod spec, irrespective of the underlying container runtime in
+  use (e.g. kata vs. runc).
+
+  *Limitation*: At present, there is no way to perform
+  runtime-specific operations that may need to occur as a result of
+  injecting a device into a container (device plugins are runtime
+  agnostic). A good example is supporting GPU passthrough
+  virtualization on kata vs. runc.
+
+- *Access to the plugin container*: When deploying a device plugin, I
+  would like to ensure that all of the operations that need to occur
+  as part of “injecting” a device into a container, also occur for the
+  “plugin container” itself.
+
+  *Limitation*: At present, there is a chicken-and-egg problem to
+  supporting this because the device plugin is the sole dictator of
+  which containers will have these operations performed on them.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+* More flexibility:
+  * Arbitrary, resource-specific setup and cleanup actions
+  * Over-the-fabric resources
+  * Custom matching of resource requests with available resources,
+    including handling of optional resource requests
+* User-friendly API for describing resource requests
+* Allow resource management plugins that can be developed and deployed
+  separately from Kubernetes and are independent of specific container
+  runtimes.
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+* An extension of the model that kube-scheduler has about
```

Comment:
These should all begin with a verb, just as all of the "Goals" bullet points do.
Also, can we put newlines between each bullet here.

Reply:
> These should all begin with a verb, just as all of the "Goals" bullet points do

So "More flexibility" and "User-friendly API" are considered verbs? Sorry, couldn't resist. :sweat_smile: 

Seriously, I am wondering whether I should also update those. I'll leave them for now and only change the non-goals because there verbs indeed work better.